<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Qcentris-396" failures="1" tests="5" name="tests.TestNGTestRunner" time="88.560" errors="0" timestamp="2025-02-15T15:42:03 EET" skipped="0">
  <testcase classname="tests.TestNGTestRunner" name="runScenario" time="10.371">
    <failure message="Expected error message: Epic sadface: Epic sadface: Password is required but got: Epic sadface: Username and password do not match any user in this service expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected error message: Epic sadface: Epic sadface: Password is required but got: Epic sadface: Username and password do not match any user in this service expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at tests.ErrorValidationTestStepDefinitions.iShouldSeeAnErrorMessage(ErrorValidationTestStepDefinitions.java:36)
at âœ½.I should see an error message "Epic sadface: Epic sadface: Password is required"(file:///C:/Users/Mahmoud.Gomaa/eclipse-workspace/Demo/src/main/resources/ErrorValidation.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="tests.TestNGTestRunner" name="runScenario" time="6.939"/>
  <system-out/>
  <testcase classname="tests.TestNGTestRunner" name="runScenario" time="6.956"/>
  <system-out/>
  <testcase classname="tests.TestNGTestRunner" name="runScenario" time="54.775"/>
  <system-out/>
  <testcase classname="tests.TestNGTestRunner" name="runScenario" time="9.519"/>
  <system-out/>
</testsuite> <!-- tests.TestNGTestRunner -->
